cmd/agent/app/integrations.go

```diff
@@ -37,6 +38,13 @@ const (
 	downloaderModule         = "datadog_checks.downloader"
 	disclaimer               = "For your security, only use this to install wheels containing an Agent integration " +
 		"and coming from a known source. The Agent cannot perform any verification on local wheels."
+	versionScript = `
```

Comment:
Do we have to run something dynamic to get the installed integration version? Can we find it statically (e.g., off disk)?

Reply:
I could manually parse the `__about__.py` file to get the version, but doing this seemed much easier and robust. What don't you like about that ?

Reply:
Well, there's just potential security issues when we run dynamic code, especially partly based on user input (the integration name). Since we sort of clean up the integration name, the risk is less here, but there is still a risk. Does this make sense?

Reply:
Is it really a concern in our case ? If the attacker is on the machine, and can execute commands as `dd-agent`, I'd say he doesn't care that he can maybe execute a python command via the agent integration subcommand, isn't that right ?

Reply:
It's always a good idea to validate user input, even if running on the same machine, and in this case we do minimally check for pattern matching (`datadog-*` which, note, could expand to deeper packages inside, e.g., `datadog_xyz.foo.bar`). Furthermore, only the `dd-agent` user should be able to execute this command, as you noted, so the scope for damage is limited.

Reply:
could we (maybe) sanitize the integration's name at least? (if that's not already the case). I'd expect an integration name to match the (case insensitive) regex `[a-z0-9_]`, right?